server/database/schema.sql

```diff
@@ -1,96 +1,110 @@
-CREATE TABLE user (
-  id INT PRIMARY KEY AUTO_INCREMENT,
-  firstname VARCHAR(80),
-  lastname VARCHAR(80),
-  password VARCHAR(80),
-  pseudo VARCHAR(80),
-  image_Profile BLOB,
-  email VARCHAR(80),
-  isAdmin BOOLEAN,
-  comment_Quantity INT,
-  ingredient_id INT
+CREATE TABLE ingredient (
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    name VARCHAR(255),
+    category VARCHAR(255),
+    image_profile BLOB,
```

Comment:
String here, you'll need to upload your images using a tool like "multer" (check it on npm website)

Reply:
Je viens de changer BLOB pour TEXT